content/html/concepts/attributes/terms/src.md

```diff
@@ -0,0 +1,48 @@
+﻿---
+Title: 'src'
+Description: 'An HTML attribute used to specify the location of a digital resource.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Attributes'
+  - 'Elements'
+  - 'Style'
+CatalogContent: 
+   - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The  **`src`** attribute is  used to specify the location of a digital source, which is typically a URL. It is required for HTML elements like [`<img>`](https://www.codecademy.com/resources/docs/html/images) so that they can appear on a website or application. 
+
+## Syntax
+
+```pseudo
+<element  src="url">
+```
+
+
+`src` can also be used in the following elements :
+
+| Direction | Description                                                                                                          |
+| --------- | -------------------------------------------------------------------------------------------------------------------- |
+| `<audio>`     | Embeds sound files on websites and applications                                                         |
+| `input`     | Creates interactive textboxes for online forms.                                                                          |
+| `<script>`    | Embeds Javascript and other executive code or data into a HTML file . |
+| `<video>`    | Embeds movie clips or other video sources into an HTML file. |
+| `<track>`    | Specifies the subtitles and closed captions for  `<audio>` and  `<video>` elements. |
+
+
+## Example #1: Image 
+The snippet below shows a logo being inserted into an HTML file:  
+```
+<img src="logo.png" alt="Codecademy logo" />
+```
+**Note: When using online images or images from the folders in an IDE workspace, always add `alt` text at the end of the `img`  element just in case the brower has trouble finding them.**
```

Comment:
```suggestion

> **Note:** When using online images or images from the folders in an IDE workspace, always add `alt` text at the end of the `<img>`  element just in case the browser has trouble finding them.
```